setup.py

```diff
@@ -0,0 +1,65 @@
+import os.path
+from setuptools import find_packages, setup
+
+# Package data
+# ------------
+_author       = 'covjson'
+_authorEmail  = 'resc@reading.ac.uk'
+_classifiers  = [
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering',
+    'Development Status :: 4 - Beta',
+    'License :: Copyright :: University of Reading',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Internet :: WWW/HTTP',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+_description  = 'Create CovJSON files from common scientific data formats'
+_downloadURL  = 'http://pypi.python.org/pypi/pycovjson/'
+_requirements = ["", "",""]
+_keywords     = ['dataset', 'coverage', 'covjson']
+_license      = 'Copyright :: University of Reading'
+_long_description    = 'A python utility library for creating CovJSON files from common scientific data formats'
+_name         = 'pycovjson'
+_namespaces   = []
+_testSuite    = 'pycovjson.test'
+_url          = 'https://github.com/Reading-eScience-Centre/pycovjson'
+_version      = '1.0.0'
+_zipSafe      = True
```

Comment:
The package is currently not zipsafe as it loads data files which are json templates, but these can probably easily be added inside python modules to avoid that issue.